src/azure-cli/setup.py

```diff
@@ -60,6 +60,7 @@
     'azure-graphrbac~=0.60.0',
     'azure-identity',
     'azure-keyvault-administration==4.0.0b3',
+    'azure-keyvault-keys~=4.4.0',
```

Comment:
Support for SKR was introduced in [`azure-keyvault-keys` 4.5.0b2](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/keyvault/azure-keyvault-keys/CHANGELOG.md#450b2-2021-08-10), and hasn't been included in a non-beta release. Would this need to be updated when track 2 support for SKR is added?

Reply:
May I know if `4.5.0b2` APIs are available for all regions and all clouds? CLI will hesitate to use beta SDK if it's not ready.